supermarq-benchmarks/supermarq/qcvv/base_experiment.py

```diff
@@ -35,66 +35,137 @@ class Sample:
     that is needed for analysis
     """
 
-    raw_circuit: cirq.Circuit
+    circuit: cirq.Circuit
     """The raw (i.e. pre-compiled) sample circuit."""
     data: dict[str, Any]
-    """The corresponding data about the circuit"""
-    probabilities: dict[str, float] | None = None
-    """The probabilities of the computational basis states"""
+    """The corresponding data about the circuit that is needed when analyzing results
+    (e.g. cycle depth)."""
+
+
+@dataclass
+class QCVVResults(ABC):
```

Comment:
probably out of scope for this pr, but imo this class would really benefit from a `_repr_pretty_`

Reply:
Good shout, I've added an issue